docs/advanced-entity-analytics/turn-on-risk-engine.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[turn-on-risk-engine]]
+== Turn on the risk scoring engine
+
+beta[]
+
+IMPORTANT: To use Entity Risk Scoring, your role must have the appropriate privileges. For more information, refer to 
+
+// <<ers-requirements, Entity Risk Scoring prerequisites>>.
+
+The latest risk scoring engine, available in {stack} version 8.11 and newer, runs hourly to aggregate `Open` and `Acknowledged` <<alerts-ui-manage, alerts>> from the last 30 days, and assigns risk score to the host or user. It then aggregates the individual risk scores and normalizes them to a 0-100 range. The engine assigns a risk level by mapping the normalized risk score to one of these levels:
+
+[width="100%",options="header"]
+|==============================================
+|Risk level |Risk score
+
+|Unknown |< 20
+|Low |20-40
+|Moderate |40-70
+|High     | 70-90
+|Critical  | > 90
+
+|==============================================
+
+[discrete]
+=== Preview risky entities
+
+In {stack} version 8.11 and newer, you can preview risky entities before installing the latest risk engine. The preview shows the riskiest hosts and users found in the 1000 sampled entities during the time frame selected in the date picker.
+
+NOTE: The preview is limited to two risk scores per {kib} instance.
+
+To preview risky entities, go to **Manage** -> **Entity Risk Score**:
+
+[role="screenshot"]
+image::images/preview-risky-entities.png[Preview of risky entities]
+
+[discrete]
+=== Turn on the latest risk engine
+
+If you're running {stack} version 8.11 or newer, and you're installing the risk scoring engine for the first time:
+
+. Go to **Manage** -> **Entity Risk Score**.
+. Turn the **Entity risk scoring** toggle on.
+
+[role="screenshot"]
+image::images/turn-on-risk-engine.png[Turn on entity risk scoring]
+
+[NOTE]
+======
+* To view risk score data, you must have alerts generated in your environment.
+* If you previously installed the original <<user-risk-score, user>> and <<host-risk-score, host risk score>> modules, and you're upgrading to {stack} version 8.11 or newer, refer to <<upgrade-risk-engine, Upgrade to the latest risk engine>>.
```

Comment:
This seems like an appropriate place to mention version #, because it's about upgrading from a previous version. But the reader needs to know this _before_ they go through the steps above, not after it.